contracts/prize-pool/PrizePool.sol

```diff
@@ -781,8 +781,8 @@ abstract contract PrizePool is PrizePoolInterface, OwnableUpgradeable, Reentranc
   /// @param _controlledToken The controlled token to add.  Cannot be a duplicate.
   function _addControlledToken(ControlledTokenInterface _controlledToken) internal {
     require(_controlledToken.controller() == this, "PrizePool/token-ctrlr-mismatch");
-    _tokens.addAddress(address(_controlledToken));
-
+    
+    _tokens.push(_controlledToken);
```

Comment:
here let's instead pass an index and set the token at that index (save 5k gas per token)